mkdocs.yml

```diff
@@ -17,7 +17,7 @@ nav:
       - tutorials/index.md
       - Getting started: tutorials/getting-started.md
       - Implement a service: tutorials/implement-service.md
-      - Implement a pipeline: tutorials/implement-pipeline.md
+      - Create a pipeline that blur faces in an image: tutorials/implement-pipeline.md
```

Comment:
Rename as "that blur**s**" and `create-a-pipeline-that-blurs-faces-in-an-image.md`.